enhancements/installer/rhel8_worker_support.md

```diff
@@ -0,0 +1,252 @@
+---
+title: rhel-8-server-worker-infra-node-support
+authors:
+  - "@mtnbikenc"
+reviewers:
+  - "@staebler"
+approvers:
+  - "@derekwaynecarr"
+creation-date: 2021-05-13
+last-updated: 2021-05-13
+status: provisional
+see-also:
+  - "https://issues.redhat.com/browse/CORS-1650"
+replaces:
+  - n/a
+superseded-by:
+  - n/a
+---
+
+# RHEL 8 Server Worker/Infra Node Support
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement proposes to add RHEL 8 Server support for deploying worker/infra
+nodes with [openshift-ansible] RHEL 8 nodes will be deployed and upgraded using
+the same Ansible playbook process however, RPM content will be delivered using
```

Comment:
The same process as what? RHEL 7? Is there a reference for that for those of us coming in between its implementation and this enhancement?

Reply:
[openshift-ansible](https://github.com/openshift/openshift-ansible) which is linked in the previous sentence is the 'Ansible playbook process' being referred to.  I will add a link to the openshift docs for adding RHEL hosts.
https://docs.openshift.com/container-platform/4.7/machine_management/adding-rhel-compute.html